src/details/ArborX_DetailsDistributedSearchTreeImpl.hpp

```diff
@@ -262,25 +262,15 @@ DistributedSearchTreeImpl<DeviceType>::sendAcrossNetwork(
                            exports.extent(7);
 
 #ifndef ARBORX_USE_CUDA_AWARE_MPI
-  (void)space;
-  auto exports_host = create_layout_right_mirror_view(exports);
-  Kokkos::deep_copy(space, exports_host, exports);
-
   auto imports_host = create_layout_right_mirror_view(imports);
 
   using NonConstValueType = typename View::non_const_value_type;
-  using ConstValueType = typename View::const_value_type;
-
-  Kokkos::View<ConstValueType *, Kokkos::HostSpace,
-               Kokkos::MemoryTraits<Kokkos::Unmanaged>>
-      export_buffer(exports_host.data(), exports_host.size());
```

Comment:
I am trying to port DTK to the latest version of ArborX (that's really a pain). There is a problem here because we allow `exports` to be of rank 8 but `doPostAndWaits` requires a `View` of rank 1. By removing the `export_buffer`, `sendAcrossNetwork` accepts a `View` that it does not support. Either we need to change the asserts in the function or undo this change (my preference). To be honest, I am not sure why the `export_buffer` was removed in the first place. Can't we have an unmanaged 1D View on CUDA?

Reply:
> Can't we have an unmanaged 1D View on CUDA?

Yes, that sounds reasonable.